### DISCUSSION TITLE
Connect to PostgresSQL

### DISCUSSION THREAD

**Discussion by Aledcode** [GreaterWMS/GreaterWMS#232] [Announcements]
Hi, I curiou to use this system with PostgresSQL, could you please give me some sample or config, that I can connected this system with PostgresSQL.

Thanks

**Comment by Singosgu** [comment]
vim greaterwms/settings.py

DATABASES = {
    'default': {
        # 'ENGINE': 'django.db.backends.sqlite3',
        # 'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'pgsql', #name
        'USER': 'postgres', #user
        "PASSWORD" : 'password', #your password
        "HOST":'',
        'PORT':5432,
    }
}

**Reply by Aledcode** [reply]
thank you

**Reply by Aledcode** [reply]
Hi I try but seems like error, it will be auto generate th table schema or not?

**Reply by Singosgu** [reply]
you should creater the schema name pgsql first, then it will auto generate the table

**Reply by Aledcode** [reply]
sorry not so clear, would you please give some example?

**Reply by Aledcode** [reply]
Ialready create the schema name "IOH_WMS"

**Reply by Aledcode** [reply]
> you should creater the schema name pgsql first, then it will auto generate the table

I already create before but still not work, the backend service restaring always when I changed to postgresql after docke compose up -d

**Reply by Singosgu** [reply]
docker container don't have postgresql
it seems

**Reply by Aledcode** [reply]
I already instaled postgressql on different container

**Reply by Singosgu** [reply]
does it run well?

**Reply by Aledcode** [reply]
> does it run well?

still not yet, could you give me example?
